### PR TITLE
adding storm_ref_data_mod.rstorm

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: docs/synapse/userguides/storm_ref_data_mod.rstorm [hunk 1]
```diff
@@ -0,0 +1,746 @@
...
+.. NOTE::
+  The specific syntax and process of node creation, modification, etc. are described in detail below. The examples here are simply meant to illustrate the autoadd and depadd concepts.
+
+*Create a node for the email address user@vertex.link. Note the secondary properties (:fqdn and :user) that are set via autoadd.*
+
+
+.. storm-cli:: [ inet:email = user@vertex.link ]
+
+
+*Create a node to represent the twitter account for The Vertex Project. Synapse creates the account itself (``inet:web:acct``) with secondary properties for ``:webpage`` (explicitly set) as well as ``:site`` and ``:user`` (via autoadd). Note the additional nodes that are created from those secondary properties via deppadd (``inet:url``, ``inet:user``, multiple FQDNs, etc.).*
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.rstorm:128]: Is this supposed to be italicized text?

File: docs/synapse/userguides/storm_ref_data_mod.rstorm [hunk 2]
```diff
@@ -0,0 +1,746 @@
...
+*Add (or modify) universal property:*
+
+
+.. storm-pre:: inet:dns:a [ .seen=("2017/08/01 01:23", "2017/08/01 04:56") ]
+::
+
+    <inet:dns:a> [ .seen=("2017/08/01 01:23", "2017/08/01 04:56") ]
+
+
+*Add (or modify) a string property to a null value:*
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.rstorm:242]: ``to a empty string value``

File: docs/synapse/userguides/storm_ref_data_mod.rstorm [hunk 3]
```diff
@@ -0,0 +1,746 @@
...
+
+.. NOTE::
+  The nodes specified by the Storm expression ( ``{ <storm> }`` ) must either already exist in the Cortex or must be created as part of the Storm expression in order for the light edges to be created.
+
+.. NOTE::
+  The query syntax used to create light edges will **yield the nodes that are inbound to the edit brackets** (that is, the nodes represented by *<query>*).
+
+**Examples:**
+
+*Link the specified FQDN and IPv4 to the media:news node referenced by the Storm expression using a "refs" light edge:*
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.rstorm:383]: supposed to be italicized?

File: docs/synapse/userguides/storm_ref_data_mod.rstorm [hunk 4]
```diff
@@ -0,0 +1,746 @@
...
+
+
+.. storm-pre:: inet:fqdn=woot.com inet:ipv4=1.2.3.4 [ <(refs)+ { media:news=a3759709982377809f28fc0555a38193 } ]
+.. storm-pre:: [inet:fqdn=newsonet.net inet:fqdn=staycools.net inet:fqdn=hugesoft.org inet:fqdn=purpledaily.com +#aka.feye.thr.apt1]
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.rstorm:387]: duplicate line?

File: docs/synapse/userguides/storm_ref_data_mod.rstorm [hunk 5]
```diff
@@ -0,0 +1,746 @@
...
+
+
+Consider the same query using edit parens inside the brackets:
+
+
+.. storm-pre:: inet:fqdn#aka.feye.thr.apt1 [-#aka.eset]
+.. storm-pre:: inet:fqdn#aka.feye.thr.apt1 [(inet:fqdn=somedomain.com +#aka.eset.thr.sednit)]
+::
+
+    inet:fqdn#aka.feye.thr.apt1 [(inet:fqdn=somedomain.com +#aka.eset.thr.sednit)]
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_data_mod.rstorm:695]: this was output as nodes in the ipynb docs

File: docs/synapse/userguides/storm_ref_data_mod.rstorm (hunk 2 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_data_mod.rstorm:242]: changed

File: docs/synapse/userguides/storm_ref_data_mod.rstorm (hunk 1 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_data_mod.rstorm:128]: it looks like this is consistent with the rest of the docs

File: docs/synapse/userguides/storm_ref_data_mod.rstorm (hunk 3 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_data_mod.rstorm:383]: yes, at least based on the prev versions

File: docs/synapse/userguides/storm_ref_data_mod.rstorm (hunk 4 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_data_mod.rstorm:387]: not a dupe, one is for pre processing and one is for printing the literal

**Pull request merged**